lib/valid_email2.rb

```diff
@@ -3,25 +3,25 @@
 require "valid_email2/email_validator"
 
 module ValidEmail2
-  BLACKLIST_FILE  = "config/blacklisted_email_domains.yml"
-  WHITELIST_FILE  = "config/whitelisted_email_domains.yml"
+  blocklist_FILE  = "config/blocklisted_email_domains.yml"
+  allowlist_FILE  = "config/allowlisted_email_domains.yml"
```

Comment:
It would be a breaking change not to still check the problematic paths (perhaps with a warning, to encourage the change away from problematic verbiage in usage)